operator/deploy/deploy-kiali-operator.sh

```diff
@@ -797,27 +818,41 @@ fi
 
 # Now deploy Kiali if we were asked to do so.
 
-if [ "${OPERATOR_INSTALL_KIALI}" != "true" ]; then
-  _BRANCH="${OPERATOR_VERSION_LABEL}"
-  if [ "${_BRANCH}" == "dev" ]; then
-    _BRANCH="master"
+print_skip_kiali_create_msg() {
+  local _ns="${OPERATOR_WATCH_NAMESPACE}"
+  if [ "${_ns}" == '""' ]; then
+    _ns="<any namespace you choose>"
   fi
   echo "=========================================="
   echo "Skipping the automatic Kiali installation."
-  echo "To install Kiali, create a Kiali custom resource in the namespace [${OPERATOR_NAMESPACE}]."
+  echo "To install Kiali, create a Kiali custom resource in the namespace [$_ns]."
   echo "An example Kiali CR with all settings documented can be found here:"
   echo "  https://raw.githubusercontent.com/kiali/kiali/${_BRANCH}/operator/deploy/kiali/kiali_cr.yaml"
```

Comment:
This URL is bad. Use:
https://github.com/kiali/kiali/blob/${_BRANCH}/master/operator/deploy/kiali/kiali_cr.yaml

Also, below.